git/meta.yaml

```diff
@@ -1,16 +1,26 @@
 package:
   name: git
-  version: 2.4.6
+  version: 2.6.3
 
 source:
-  git_url: git@github.com:git/git.git
-  git_tag: v2.4.6
+  fn: v2.6.3.tar.gz
+  url: https://github.com/git/git/archive/v2.6.3.tar.gz
+  md5: 65f659fb55ba49f17de03d1d9a859a83
+
+build:
+  # git hardcodes paths to external utilities (e.g. curl)
+  detect_binary_files_with_prefix: true
 
 requirements:
   build:
+    - gcc
+    - expat
+    - curl
     - openssl
     - zlib
   run:
+    - expat
+    - curl
```

Comment:
@ccordoba12 I've just `ldd`ed and `grep`ed the binaries, and I couldn't find `libgcc` dependency there.